Tests/SonarScanner.MSBuild.Tasks.IntegrationTest/TargetsTests/RoslynTargetsTests.cs

```diff
@@ -201,15 +205,22 @@ public void Settings_ValidSetup_NonLegacyServer_MergeSettings()
 
   <ItemGroup>
     <!-- all analyzers specified in the project file should be preserved -->
-    <Analyzer Include='c:\original\should.be.removed\analyzer1.dll' />
+    <Analyzer Include='c:\original\should.be.preserved\analyzer1.dll' />
     <Analyzer Include='original\should.be.preserved\analyzer3.dll' />
+    <Analyzer Include='should.be.preserved.SonarAnalyzer.Fake.dll' />
+    <Analyzer Include='c:\SonarAnalyzer\should.be.preserved.SomeAnalyzer.dll' />
+    <Analyzer Include='c:\should.be.removed\SonarAnalyzer.CFG.dll' />
+    <Analyzer Include='c:\should.be.removed\SonarAnalyzer.dll' />
+    <Analyzer Include='c:\should.be.removed\SonarAnalyzer.CSharp.dll' />
+    <Analyzer Include='c:\should.be.removed\SonarAnalyzer.vIsUaLbAsIc.dll' />
+    <Analyzer Include='c:\should.be.removed\SonarAnalyzer.Security.dll' />
   </ItemGroup>
   <ItemGroup>
-    <!-- These additional files don't match ones in the config and should be preserved -->
-    <AdditionalFiles Include='should.not.be.removed.additional1.txt' />
-    <AdditionalFiles Include='should.not.be.removed.additional2.txt' />
+    <!-- These additional files should be preserved -->
+    <AdditionalFiles Include='should.be.preserved.additional1.txt' />
+    <AdditionalFiles Include='should.be.preserved.additional2.txt' />
 
-    <!-- This additional file matches one in the config and should be replaced -->
+    <!-- This additional file matches one in the config but will not be removed as only duplicate SonarAnalyzer libraries are removed  -->
```

Comment:
revert change to comment? _(seems it's out of date, as these are still removed)_